osu.Game/Updater/SimpleUpdateManager.cs

```diff
@@ -20,44 +15,48 @@ namespace osu.Game.Updater
     /// An update manager that shows notifications if a newer release is detected.
     /// Installation is left up to the user.
     /// </summary>
-    public partial class SimpleUpdateManager : UpdateManager
+    /// <remarks>
+    /// Implementers are expected to set <see cref="Version"/> to something that matches their release versioning/tagging system.
+    /// </remarks>
```

Comment:
This is a sign of bad design and needs to be handled better. Constructor, abstract property, whatever, but consumers should be _forced_ to set this - you shouldn't need to rely on a loose setter only mentioned in passing in the remarks section.